addons/mrp/models/mrp_bom.py

```diff
@@ -198,6 +198,26 @@ def create(self, vals_list):
             production._link_bom(res[0])
         return res
 
+    def write(self, vals):
+        res = super().write(vals)
+        relevant_fields = ['bom_line_ids', 'byproduct_ids', 'product_tmpl_id', 'product_id', 'product_qty', 'operation_ids']
```

Comment:
I think we have, because if you change the `product_qty` but don't change the BoM's qty, the MO won't be notified than the BoM was updated. But even if the BoM's line qty didn't change, the ratio isn't the same anymore (if you need 2 components for 2 finish products, then modify the BoM for 1 qty, the MO using this BoM now need two times more components than before.)